components/level/game.tsx

```diff
@@ -15,12 +16,13 @@ import { LevelContext } from '../../contexts/levelContext';
 import { PageContext } from '../../contexts/pageContext';
 import BlockState from '../../models/blockState';
 import Control from '../../models/control';
-import Level from '../../models/db/level';
+import Level, { EnrichedLevel } from '../../models/db/level';
 import Move from '../../models/move';
 import Position, { getDirectionFromCode } from '../../models/position';
 import SquareState from '../../models/squareState';
 import GameLayout from './gameLayout';
 
+export const USER_BEST_MOVE_CHECKPOINT_SLOT = 10;
```

Comment:
All the checkpoint changes are here, will wait for the other PR to merge or for the checkpoint code to be removed before reviewing